extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/AccessLogHandler.java

```diff
@@ -145,7 +149,16 @@ public void handle(RoutingContext rc) {
         QuarkusRequestWrapper.get(rc.request()).addRequestDoneHandler(new Handler<Void>() {
             @Override
             public void handle(Void event) {
+                Map<String, Optional<String>> serialized = ((ExchangeAttributeSerializable) tokens).serialize(rc);
+                for (Map.Entry<String, Optional<String>> entry : serialized.entrySet()) {
+                    MDC.put(entry.getKey(), entry.getValue().orElse("-"));
```

Comment:
Wouldn't it be better to skip entries without values so they are not included in the logged JSON?

Reply:
This was done in order to preserve the same behavior as is in "message" entry, where all parameters are concatenated and missing values are replace with "-".